components/role-mgt/org.wso2.carbon.identity.role.v2.mgt.core/src/main/java/org/wso2/carbon/identity/role/v2/mgt/core/dao/SQLQueries.java

```diff
@@ -414,4 +414,20 @@ public class SQLQueries {
     public static final String INSERT_MAIN_TO_SHARED_ROLE_RELATIONSHIP = "INSERT INTO UM_SHARED_ROLE " +
             "(UM_SHARED_ROLE_ID, UM_MAIN_ROLE_ID, UM_SHARED_ROLE_TENANT_ID, UM_MAIN_ROLE_TENANT_ID) " +
             "VALUES (:UM_SHARED_ROLE_ID;, :UM_MAIN_ROLE_ID;, :UM_SHARED_ROLE_TENANT_ID;, :UM_MAIN_ROLE_TENANT_ID;)";
+
+    public static final String GET_NOT_RESTRICTED_USERNAMES_BY_ROLE_HEAD =
+            "SELECT UM_USER_NAME FROM UM_HYBRID_USER_ROLE WHERE UM_ROLE_ID = :roleId; AND UM_USER_NAME IN (";
```

Comment:
```suggestion
            "SELECT UM_USER_NAME FROM UM_HYBRID_USER_ROLE WHERE UM_ROLE_ID = :roleId; AND UM_USER_NAME IN (";
```
For the places like ": roleId" can't we use the defined constants?

Reply:
Addressed in commit: [4a2588d](https://github.com/wso2/carbon-identity-framework/pull/6303/commits/4a2588d7b3e0358ae048d456962756250c9b0577)